flatpak_builder_lint/appstream.py

```diff
@@ -12,8 +12,33 @@
 def validate(path: str) -> dict:
     if not os.path.isfile(path):
         raise FileNotFoundError("AppStream file not found")
+
+    overrides = {
+        "all-categories-ignored": "error",
```

Comment:
This one doesn't exist on a release yet https://github.com/ximion/appstream/commit/0653448a469e3b61b4b5b54ca13eca7a92b65dd1

Reply:
Good catch, we can backport it.